### DISCUSSION TITLE
Does budibase support navigation sub menu?

### DISCUSSION THREAD

**Discussion by imfht** [Budibase/budibase#8306] [Help]
eg:
![image](https://user-images.githubusercontent.com/15059493/196399198-ad0b795d-e7c9-4060-b300-48455f79f67d.png)

**Comment by andz-bb** [comment]
Hey @imfht, this isn't currently possible natively in Budibase but there is an open issue for it here: #6659 

It's worth commenting your interest in that issue if this is something you would like to see developed. As a workaround this is something that could possibly be developed as a custom component plugin for Budibase, although it can't be integrated with the built-in Budibase navigation bar.

**Reply by mmays77** [reply]
I would like to be able to use a feature like this in my Budibase applications.
